common/protob/messages-management.proto

```diff
@@ -191,7 +191,14 @@ message ResetDevice {
     optional uint32 u2f_counter = 7;                    // U2F counter
     optional bool skip_backup = 8;                      // postpone seed backup to BackupDevice workflow
     optional bool no_backup = 9;                        // indicate that no backup is going to be made
-    optional bool slip39 = 10;                          // indicate that we want SLIP-39 backup not BIP-39
```

Comment:
just a question: we never released anything that knows about this field, right? otherwise changing type would not be nice

Reply:
We did not - only the internal fw release last month had it, but that wasn't public.